sormas-backend/src/test/java/de/symeda/sormas/backend/TestDataCreator.java

```diff
@@ -160,7 +160,7 @@ private void createUserRoles() {
 			userRoleDto.setEmailNotificationTypes(defaultUserRole.getEmailNotificationTypes());
 			userRoleDto.setSmsNotificationTypes(defaultUserRole.getSmsNotificationTypes());
 			userRoleDto.setJurisdictionLevel(defaultUserRole.getJurisdictionLevel());
-			userRoleDto = beanTest.getUserRoleFacade().saveUserRole(userRoleDto);
```

Comment:
Please keep using UserRoleFacade.saveUserRole. To make it work add a null check for the currentUserName in CurrentUserService:45